Pod/Classes/VOKCoreDataCollectionTypes.h

```diff
@@ -0,0 +1,61 @@
+//
+//  VOKCoreDataCollectionTypes.h
+//  Pods
+//
+//  Created by Carl Hill-Popper on 10/23/15.
+//
+//
+
+#ifndef VOKCoreDataCollectionTypes_h
```

Comment:
Is `typedef`ing collection types common? I don't know no fancy Swift or modern Objective-C.
it's very cool nonetheless